packages/rum/src/domain/rumEventsCollection/resource/resourceCollection.spec.ts

```diff
@@ -208,6 +209,198 @@ describe('resourceCollection', () => {
   })
 })
 
+describe('resourceCollection V2', () => {
+  let setupBuilder: TestSetupBuilder
+
+  afterEach(() => {
+    setupBuilder.cleanup()
+  })
```

Comment:
I would keep the `afterEach` declarations after their respective `beforeEach` declaration to help comprehension.